### ISSUE TITLE
GCC/Clang Warning unused variable

### ISSUE THREAD

**Issue by DanRStevens** [ldicker83/OPHD#129]
GCC (and Clang) warns about an unused variable:
```
src/States/MapViewStateTurn.cpp: In member function 'void MapViewState::nextTurn()':
src/States/MapViewStateTurn.cpp:322:6: warning: unused variable 'result' [-Wunused-variable]
  int result = pather->Solve(mTileMap->getTile(299, 149), mTileMap->getTile(10, 10), &path, &totalCost);
      ^~~~~~
```

**Comment by DanRStevens** [comment]
Odd not simply because the result is unused, but also because it uses hardcoded `x`, `y` values. I'm guessing this is some sort of test code.

**Comment by ldicker83** [comment]
It is. This is the A* code I was using to test path generation for truck routing.

**Comment by DanRStevens** [comment]
This was closed by #168.
